### PR TITLE
Add documentation for updated GCR plugin

### PR DESCRIPTION
**Pull request by tonglil**
Pending https://github.com/drone-plugins/drone-docker/pull/144.

### CODE REVIEW

File: content/drone-plugins/drone-gcr/index.md [hunk 1]
```diff
@@ -8,6 +8,73 @@ logo: google_gcr.svg
 image: plugins/gcr
 ---
 
-{{% alert error %}}
-We are looking for volunteers to port this plugin to drone 0.5
+{{% alert info %}}
```
Comment (tonglil) [inline content/drone-plugins/drone-gcr/index.md]: Not sure if this is a valid use of this.

File: content/drone-plugins/drone-gcr/index.md [hunk 2]
```diff
@@ -8,6 +8,73 @@ logo: google_gcr.svg
...
+The GCR plugin is an extension around the Docker plugin. Please see the documentation for the Docker plugin for other configuration options.
```
Comment (tonglil) [inline content/drone-plugins/drone-gcr/index.md]: Not sure if I should hard link to the Docker plugin docs, or just let people find it.

File: content/drone-plugins/drone-gcr/index.md (hunk 1 above)
Comment (tboerger) [reply content/drone-plugins/drone-gcr/index.md]: This is not valid anymore since it's managed within drone-plugins/drone-docker now.

Comment (tboerger) [reply content/drone-plugins/drone-gcr/index.md]: Scratch that, I was just talking about the deleted sentence.

File: content/drone-plugins/drone-gcr/index.md (hunk 2 above)
Comment (tboerger) [reply content/drone-plugins/drone-gcr/index.md]: You should link with the help of Hugo links, not a hard link :)

File: content/drone-plugins/drone-gcr/index.md (hunk 1 above)
Comment (tonglil) [reply content/drone-plugins/drone-gcr/index.md]: Maybe an informational alert box can point people to the additional options that drone-docker provides?

**Pull request merged**